.github/workflows/uplift-merges.yml

```diff
@@ -11,28 +11,41 @@ on:
 permissions:
   contents: read
 
+environment: botmobile
```

Comment:
Is shippable_builds missing the botmobile environment?

Reply:
It is not. shippable_builds only needs botmobile in certain environments, but it does need them in places where there is already an existing one, like `thunderbird_beta`. There can only be one environment. So I need to add the keys to each of these environments as well. This is what I was alluding to in my initial comments.

Reply:
That's right. For now at least I think we should keep separate environments.